CONTRIBUTING.md

```diff
@@ -1,3 +1,3 @@
 # CONTRIBUTING
 
-See [How to contribute](https://docs.nebula-graph.io/manual-EN/4.contributions/how-to-contribute/) for details about how to contribute to **Nebula Graph**.
+See [How to contribute](https://docs.nebula-graph.io/2.0.1/15.contribution/how-to-contribute/) for details about how to contribute to **Nebula Graph**.
```

Comment:
I personally don't think it is a good idea to add a link with version number because new versions come out from time to time, meaning that you have to update the link from time to time. 
We have a how-to-contribute file in the nebula-community repo and here is the link: https://github.com/vesoft-inc/nebula-community/blob/master/Contributors/how-to-contribute.md. I think this is a better choice.

Reply:
Agreed, this is a better choice, will replace it with this link instead.

Reply:
done